### ISSUE TITLE
WebView?

### ISSUE THREAD

**Issue by Sznymo** [webui-dev/nim-webui#36]
Does WebUI-Nim already support WebView?

**Comment by Sznymo** [comment]
Error: attempting to call undeclared routine: 'showWv'
In the latest version after importing 
import webui

**Comment by neroist** [comment]
> Error: attempting to call undeclared routine: 'showWv' In the latest version after importing import webui

use `webui#head`

**Comment by Sznymo** [comment]
![zrzut_ekranu_20240713_224617](https://github.com/user-attachments/assets/6bde94f0-3a2f-4ffd-b2ff-97e259842c86)
![zrzut_ekranu_20240713_224853](https://github.com/user-attachments/assets/ad4068b0-7fe3-413f-a5d0-2104c1d815f5)
error continues to occur, in addition nimble install installs versions from the neroist repository

**Comment by Sznymo** [comment]
good I see where there is an error "nimble install webui and nimble install https://github.com/webui-dev/nim-webui" they install the old version by some miracle

**Comment by ITwrx** [comment]
trying to install latest code, but `nimble install webui@#head` fails with: `Version may only consist of numbers and the '.' character but found '-'.`

**Comment by neroist** [comment]
@ITwrx try installing now, it should be fixed. (also seems like you should've opened a new issue for this?)

**Comment by ITwrx** [comment]
@neroist It works now, thanks!  I was just trying to be "low key" with only a comment instead of an issue. I figured i would just comment, and if anyone saw it and wanted to address it great, otherwise i would just wait on 2.5.0 (non beta) to have a tagged release. I wasn't in a huge hurry.
